src/Microcosm.js

```diff
@@ -55,11 +55,17 @@ Microcosm.prototype = {
     return this
   },
 
+  shouldTransactionMerge() {
+    return true
+  },
```

Comment:
This will save the last 500 transactions, on the 501st, the first transaction will be merged in; working from left to right.

Actually, I will add test coverage for specifically this example.